tests/unit/test_checkpointing.py

```diff
@@ -0,0 +1,93 @@
+from unittest.mock import MagicMock
+from unittest import TestCase
+from fenics import FunctionSpace, UnitSquareMesh, Expression, interpolate
+from fenicsprecice.solverstate import SolverState
+
+
+class TestCheckpointing(TestCase):
+    def test_solverstate_basic(self):
+        """
+        Check if correct values are read from the checkpoint, while the state of the object that is copied is not changed
+        """
+        time = 0.5
+        iteration = 1
+        size = 5
+        mesh = UnitSquareMesh(size,size)
+        V = FunctionSpace(mesh, 'P', 2)
+        E = Expression("t", t=time, degree=2)
```

Comment:
Why use `time` here? I think this makes it unnecessarily hard to read. Why not the following:

```suggestion
        dummy_value = 42
        E = Expression("x", x=dummy_value, degree=2)
```